README.md

```diff
@@ -319,6 +319,19 @@ This can be useful when developing. If the config package is a dependency of you
 and you do not need to create a merge plan file when developing your package.
 For example, this is implemented in [yiisoft/yii-runner](https://github.com/yiisoft/yii-runner).
 
+### `package-types`
+
+The `package-types` option define package types for process by composer plugin. By default, it is "library" and
+"composer-plugin". You can add custom types:
+
+```json
+"extra": {
+    "config-plugin-options": {
+        "package-types": ["library", "my-extension"]
```

Comment:
This add new types, not override? So should remove "library"?

Reply:
This override default value. May be will case, when need process only packages with specified type.

Reply:
So it should be "You can **set** / **override** custom types:".
"add" will be confusing?

Reply:
Fixed